src/main/java/com/jagrosh/jmusicbot/utils/QueueSupplier.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I think this file could be moved to the queue folder, since it's only ever used by queue-related things

Reply:
done